docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-lsass-memory-dump-creation.asciidoc

```diff
@@ -0,0 +1,79 @@
+[[prebuilt-rule-0-16-2-lsass-memory-dump-creation]]
+=== LSASS Memory Dump Creation
+
+Identifies the creation of a Local Security Authority Subsystem Service (lsass.exe) default memory dump. This may indicate a credential access attempt via trusted system utilities such as Task Manager (taskmgr.exe) and SQL Dumper (sqldumper.exe) or known pentesting tools such as Dumpert and AndrewSpecial.
```

Comment:
```suggestion
Identifies the creation of a Local Security Authority Subsystem Service (LSASS) default memory dump. This may indicate a credential access attempt via trusted system utilities such as Task Manager (taskmgr.exe) and SQL Dumper (sqldumper.exe) or known pentesting tools such as Dumpert and AndrewSpecial.
```

Reply:
Mainly making this edit to align it with the way the acronym is presented in `prebuilt-rule-0-16-2-lsass-memory-dump-handle-access.asciidoc`.